tools/clang/test/DXC/print-passes.hlsl

```diff
@@ -0,0 +1,16 @@
+// RUN: %dxc -E main -T vs_6_0 %s -print-before-all 2>&1 | FileCheck -check-prefix=BEFORE1 --check-prefix=BEFORE2 %s
+// RUN: %dxc -E main -T vs_6_0 %s -print-after-all 2>&1 | FileCheck -check-prefix=AFTER1 --check-prefix=AFTER2 %s
+// RUN: %dxc -E main -T vs_6_0 %s -print-before-all -print-after-all 2>&1 | FileCheck -check-prefix=BEFORE1 -check-prefix=BEFORE2 -check-prefix=AFTER1 --check-prefix=AFTER2 %s
+// RUN: %dxc -E main -T vs_6_0 %s -print-before verify 2>&1 | FileCheck -check-prefix=BEFORE1 %s
+// RUN: %dxc -E main -T vs_6_0 %s -print-after hlsl-dxilemit 2>&1 | FileCheck -check-prefix=AFTER2 %s
```

Comment:
Can we have a check for multiple `-print-after`s?

Reply:
and `print-before`s . . .

Reply:
done